src/db/schema.sql

```diff
@@ -0,0 +1,28 @@
+-- comapany database holding the employee, role and department tables
+
```

Comment:
Your schema is missing the actual creation of the database. This needs to be added in before creating the tables or this does not work.
```
DROP DATABASE IF EXISTS company_db;
CREATE DATABASE company_db;

USE company_db;
```